### DISCUSSION TITLE
Copying version of app needs pen and paper

### DISCUSSION THREAD

**Discussion by jidanni** [community/community#2823] [Mobile]
Cannot copy version of app with just touching.
 Need  a pencil and paper or a screenshot just to copy the version.
![Screenshot_20201202-220944](https://user-images.githubusercontent.com/1230959/100883374-3967e080-34eb-11eb-9043-04587b172497.jpg)

**Comment by candyho** [comment]
Hey thanks for your feedback. I've created an issue in our internal repository to follow up on this.
